.github/workflows/publish-alpha.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Alpha
+on:
+  workflow_dispatch:
+    inputs:
+      preid:
+        description: 'Preid used to publish package. Must be unique per branch.'
+        required: true
+        default: 'alpha'
+      ref:
+        description: 'Commit to deploy from. Defaults to branch used for workflow_dispatch action.'
+        required: false
+        default: ''
+jobs:
+  publish-alpha:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: '0' # Need the history to properly select the canary version number
+          ref: ${{ github.event.inputs.ref }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build packages
+        run: npm run build
+      - run: ./node_modules/.bin/lerna publish --canary --force-publish=\* --preid ${{ github.event.inputs.preid }} --dist-tag canary --yes
```

Comment:
I think we don't want a `force-publish` here.
We should probably also just use independent versioning for the plugins... since I'm hoping they stabilize, and therefore don't need to be updated for a while.

Reply:
Ya good point about force publish.

I set up lerna in independent versioning already when I added lerna to the repo